images/dind/master/openshift-sdn-master-setup.sh

```diff
@@ -34,10 +34,13 @@ function openshift-sdn-setup() {
     /usr/local/bin/oc --config="${kube_config}" adm policy add-scc-to-user anyuid -z openshift-sdn
   fi
 
-  os::util::wait-for-condition "kubernetes token" "ensure-token ${config_dir} ${kube_config}" "120"
+  os::util::wait-for-condition "kubernetes token" "ensure-token ${config_dir}/master ${kube_config}" "120"
+  # Launch the network process
+  exec openshift-sdn-plugin --config=${config_dir}/node-openshift-master-node/node-config.yaml --kubeconfig="${kube_config}" --loglevel=${DEBUG_LOGLEVEL:-4}
```

Comment:
Doesn't the master also run openshift-sdn-node.service ? It seems like this shouldn't be necessary here